inst/shiny/ui.R

```diff
@@ -20,7 +20,7 @@ shinyUI(fluidPage(
       checkboxInput('arg_assign', 'Replace = with <-', FALSE),
       checkboxInput('arg_brace', 'Put { on a new line', FALSE),
       numericInput ('arg_indent', 'Number of spaces for indentation', 4, min = 0),
-      numericInput ('arg_width', 'Minimum line width', 70, min = 20, max = 500),
+      numericInput ('arg_width', 'Maximum line width', 70, min = 20, max = 500),
```

Comment:
I guess that is equally confusing... "Minimum line width" is technically correct, although users won't really understand it unless they read the documentation.

This is an interactive app, so they can change the value and guess what it means through interactions if they don't want to read the docs.